notebook/notebookapp.py

```diff
@@ -264,6 +264,8 @@ def init_settings(self, jupyter_app, kernel_manager, contents_manager,
 
             # Jupyter stuff
             started=now,
+            # place for extensions to register activity
+            activity_sources={},
```

Comment:
'sources' suggests to me that it contains something like functions you call to get activity timestamps. Maybe call it something like `last_activity_times`?

Reply:
Ping @minrk . I can do the name change if you're OK with it.

Reply:
Sorry, just got back to it